tools/Customization/DevHome.Customization/Strings/en-us/Resources.resw

```diff
@@ -229,17 +257,17 @@
     <value>Show hidden and system files</value>
     <comment>The header for the show hidden and system files setting.</comment>
   </data>
-  <data name="DevDriveOptimizationSuggestion" xml:space="preserve">
-    <value>Suggestion</value>
-    <comment>Dev drive optimization suggestion</comment>
+  <data name="UntrustedDevDriveDescription" xml:space="preserve">
+    <value>Mircosoft Defender performance mode is only available for Trusted Dev Drive volumes </value>
```

Comment:
Should "trusted" be capitalized? Typo in Microsoft.
```suggestion
    <value>Microsoft Defender performance mode is only available for trusted Dev Drive volumes</value>
```

Reply:
FIgma has "Trusted". Confirming with design.